cmd/snap/cmd_run.go

```diff
@@ -75,10 +75,11 @@ type cmdRun struct {
 	// This options is both a selector (use or don't use strace) and it
 	// can also carry extra options for strace. This is why there is
 	// "default" and "optional-value" to distinguish this.
-	Strace    string `long:"strace" optional:"true" optional-value:"with-strace" default:"no-strace" default-mask:"-"`
-	Gdb       bool   `long:"gdb"`
-	Gdbserver string `long:"experimental-gdbserver" default:"no-gdbserver" optional-value:":0" optional:"true"`
-	TraceExec bool   `long:"trace-exec"`
+	Strace                string `long:"strace" optional:"true" optional-value:"with-strace" default:"no-strace" default-mask:"-"`
+	Gdb                   bool   `long:"gdb"`
+	Gdbserver             string `long:"gdbserver" default:"no-gdbserver" optional-value:":0" optional:"true"`
+	ExperimentalGdbserver string `long:"experimental-gdbserver" default:"no-gdbserver" optional-value:":0" optional:"true" hidden:"yes"`
```

Comment:
go-flags seems to have some aliasing support, though the docs states it works on commands https://pkg.go.dev/github.com/jessevdk/go-flags#hdr-Available_field_tags but maybe worth trying